src/dsp_tools/commands/xmlupload/check_consistency_with_ontology.py

```diff
@@ -22,30 +22,30 @@ def do_xml_consistency_check_with_ontology(onto_client: OntologyClient, root: et
     This function takes an OntologyClient and the root of an XML.
     It retrieves the ontologies from the server.
     It iterates over the root.
-    If it finds any invalid properties or classes, they are printed out and a UserError is raised.
+    If it finds any invalid properties or classes, they are printed out and a InputError is raised.
 
      Args:
          onto_client: client for the ontology retrieval
          root: root of the XML
 
      Raises:
-         UserError: if there are any invalid properties or classes
+         InputError: if there are any invalid properties or classes
     """
     onto_check_info = ProjectOntosInformation(
         default_ontology_prefix=onto_client.default_ontology,
         onto_lookup=onto_client.get_all_ontologies_from_server(),
     )
     classes_in_data, properties_in_data = _get_all_classes_and_properties_from_data(root)
-    _find_if_all_classes_and_properties_exist_in_onto(classes_in_data, properties_in_data, onto_check_info)
+    _find_all_classes_and_properties_exist_in_onto(classes_in_data, properties_in_data, onto_check_info)
 
 
-def _find_if_all_classes_and_properties_exist_in_onto(
+def _find_all_classes_and_properties_exist_in_onto(
```

Comment:
For me, this name grammatically not clear. It says: "Find all classes and properties exist in onto". The conjugated "exist" does not fit into this sentence.

Perhaps you mean one of the following:

- Find all classes and properties in the onto
- Check all classes and properties

Reply:
Well for me the "if" clause was no problem, then it was clear...
I am not sure however, if "check" is a good choice, because we not only check it but find the problems and return those.

Reply:
So if you had to formulate 1 sentence that best describes what this function does, what would it be? If we restrain ourselves to 1 main sentence, without subclause, then I would vote for "Check all classes and properties". 

Of course, it also finds problems and returns them, but perhaps that's too much to pack into 1 function name.